casts/countryPicker/src/UICountryPicker/CountryPicker.tsx

```diff
@@ -104,6 +107,10 @@ export function CountryPicker({
     }, [search]);
 
     React.useEffect(() => {
+        // We should use react-native-android-keyboard-adjust
+        // to fix android keyboard work with UIBottomSheet
+        isAndroid && AndroidKeyboardAdjust.setAdjustNothing();
```

Comment:
Well, I've tested it. 

```
AndroidKeyboardAdjust.setAdjustNothing()
```
The script above makes the keyboard `not adjusted` throw all the screens.

And when I close this modal and open the chat, they keyboard is not adjusted (as RESIZE) anymore.

Consider returning the following function on unmount with some proper comment
```
() => isAndroid && AndroidKeyboardAdjust.setAdjustResize();
```

Reply:
Actually it's better to find the previous state and return it on unmount!

Reply:
@sertony Yes, you are right. I rewrote this part, but not onMount/unMount, but when changing `visible`, this is more suitable in this case.